languages/cpp/exercises/concept/numbers/.docs/after.md

```diff
@@ -0,0 +1,43 @@
+### Special integer types
+
+There are two other important integer types often used in the standard library:
```

Comment:
It looks like the `after.md` document adds to what is listed in the `introduction.md` document. As per [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#docsaftermd), this file should at least contain all information that is introduced in the `introduction.md` document.